zkevm-circuits/src/evm_circuit/execution/callop.rs

```diff
@@ -512,6 +602,17 @@ impl<F: Field> ExecutionGadget<F> for CallOpGadget<F> {
     }
 }
 
+#[inline]
+fn is_opcode<F: Field>(opcode: &Cell<F>, opcode_id: OpcodeId) -> Expression<F> {
+    generate_lagrange_base_polynomial(
+        opcode,
+        opcode_id.as_u8() as usize,
+        [OpcodeId::CALL, OpcodeId::DELEGATECALL, OpcodeId::STATICCALL]
+            .into_iter()
+            .map(|op_id| op_id.as_u8() as usize),
```

Comment:
If this fn is only for CALL-related opcodes, and you just generate a lagrange poly for the 3 options, the `is_opcode` name of the function is a really bad one IMO. 
And 2, if we rename the fn to: `is_call_related_opcode`, we can have this as a fn associated to a `Cell` directly. And so, pass less stuff as parameter.

Also, unsure why do we enforce the `inline`. Have you tested this to verify that it's worth it?

Reply:
Deleted this useless function `is_opcode`.

I am sorry that I missed to delete it before. @icemelon also mentioned it in https://github.com/privacy-scaling-explorations/zkevm-circuits/pull/847#discussion_r1025947817. It has already been replaced with `IsZeroGadget`.